devdoc/CodeSubmissions.md

```diff
@@ -27,7 +39,25 @@ There are a few things that should be considered when creating a PR to increase
   - Check subtractions are the right way around, and potential for overflow.
   - New files with the wrong copyright date
 * Check the target branch (see below)
-* Request one or more reviews.  For relatively simple changes a single reviewer is normally enough.
+* Request one or more reviewers.  For relatively simple changes a single reviewer is normally enough.
+* For more suggestions, please refer to https://github.com/hpcc-systems/HPCC-Platform/wiki/Git-commit-message-guidelines 
+* examples of Pull Requests that can be improved\
+  - HPCC-32193 Fix some issues with spill stats in smart join activity 
+    - This could mention affected component.
+    - If it was only affecting a single file (which this PR was not), then it could mention that as well.
```

Comment:
I am not sure there is value in mentioning the file in the commit - since that isn't something the user is generally interested in, and it is easy to derive from the commit.

Reply:
@ghalliday   I've updated the commit message.  let me know if that works.  If anything, I could use this as an example instead of what I put.